examples/resources/flagsmith_environment/import.sh

```
@@ -0,0 +1 @@
+terraform import flagsmith_environment.some_environment <environment_uuid>
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this simple command.

(SC1009)